client/search.js

```diff
@@ -0,0 +1,22 @@
+let url = document.querySelector(".url");
+let title = document.querySelector(".title");
+let desc = document.querySelector(".desc");
+let tags = document.querySelector(".tags");
+
+let search = localStorage.getItem("searchValue");
+
+const getResult = async () => {
+  const response = await fetch("http://localhost:5500/search");
+  const data = await response.json();
+  for (let i = 0; i < data.length; i++) {
```

Comment:
For loop not needed as would be applied to one element

Reply:
We are using the for loop for the data array in order check where the value of the input text